frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+export function Login() {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const handleLogin = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+
+        fetch('http://localhost:8080/login', {
+            method: 'POST',
+            headers: {
+                'Accept': 'application/json, text/plain, */*',
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({ email, password })
+        })
+        .then(async res => {
+            // Ellenőrizd a válasz státuszát
+            if (!res.ok) {
+                throw new Error(`HTTP error! status: ${res.status}`);
+            }
+
+            // Próbáld meg parse-olni a választ
+            const data = await res.json().catch(() => {
+                throw new Error('Invalid JSON response');
+            });
+
+            // Ha minden rendben, logold ki a választ
```

Comment:
Well one thing about comments: if you somehow try to make logical separation with comments, those might have been functions too, which are sometimes better choices. Not always.